### PR TITLE
chore: add cncf project promotion

### PR DESCRIPTION
**Pull request by YiniXu9506**
Signed-off-by: YiniXu9506 <xuyini@pingcap.com>

Add section for cncf graduate promotion

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -28,19 +22,13 @@
...
       </p>
 
-      <!-- <br />
-
-      <div class="notification has-background-white is-radiusless">
-        <p class="title is-size-3 is-size-4-mobile has-text-weight-light">
-          TiKV is now a <a href="https://cncf.io" target="_blank">Cloud Native Computing Foundation</a> incubating project! 🎉🍾
+      <div class="notification is-radiusless">
+        <p class="title is-size-3 is-size-4-mobile">
+          TiKV is a <a href="https://cncf.io" target="_blank" class="is-link">Cloud Native Computing Foundation</a> graduated project
```
Comment (dcalvin) [inline layouts/index.html:27]:
```suggestion
          We are a <a href="https://cncf.io" target="_blank" class="is-link">Cloud Native Computing Foundation</a> graduated project.
```

Comment (dcalvin) [reply layouts/index.html:27]: Can we add some cheering emojis? @YiniXu9506

Comment (YiniXu9506) [reply layouts/index.html:27]: Yes. How about this? 🎉

Comment (dcalvin) [reply layouts/index.html:27]: Looks good!

**Review by dcalvin — APPROVED** [review]
LGTM

**Pull request merged**